packages/components/layout/src/Layout.styles.ts

```diff
@@ -12,16 +12,20 @@ export const getLayoutMaxWidthStyles = (variant: LayoutProps['variant']) => {
 
   return css({
     width: '100%',
-    maxWidth: '1550px',
-    padding: `0 ${tokens.spacingXs}`,
+    maxWidth: '1524px',
+    boxShadow:
+      '0px 6px 16px -2px rgba(25, 37, 50, 0.1), 0px 3px 6px -3px rgba(25, 37, 50, 0.15)',
+    borderRadius: `10px 10px 0 0`,
+    margin: '12px 16px 0',
```

Comment:
I think we need to hide overflow on this element to ensure child elements get cut by the border radius.

Reply:
```css
  '& > *': {
      borderRadius: `10px 10px 0 0`,
    },
```

?

Reply:
I don't think we'll have a reliable way to do this beyond ensuring the child component is given adequate spacing so that it doesn't clip the rounded corners.